### ISSUE TITLE
On 1.21.0 cri-o fails to start because of incorrect SELinux check

### ISSUE THREAD

**Issue by Elettronik** [cri-o/cri-o#5097]
**Description**

CRI-O version 1.21.1 Fails to start because of an incorrect SELinux check

**Steps to reproduce the issue:**
1. Start crio
2.
3.

**Describe the results you received:**
It fail to start

**Describe the results you expected:**
It started

**Additional information you deem important (e.g. issue happens only occasionally):**

Output of start
```
INFO[2021-07-17 11:53:31.852998073Z] Starting CRI-O, version: 1.21.0, git: bc1ef35a932acc2f6f3b6d3eb19a4f68aa9423f6(dirty) 
INFO[2021-07-17 11:53:31.853347428Z] Node configuration value for hugetlb cgroup is true 
INFO[2021-07-17 11:53:31.853458986Z] Node configuration value for pid cgroup is true 
INFO[2021-07-17 11:53:31.854167436Z] Node configuration value for memoryswap cgroup is true 
INFO[2021-07-17 11:53:31.860911621Z] Node configuration value for systemd CollectMode is true 
DEBU[2021-07-17 11:53:31.861982925Z] [graphdriver] trying provided driver "overlay"  file="drivers/driver.go:252"
DEBU[2021-07-17 11:53:31.933535724Z] Overlay test mount with multiple lowers failed invalid argument  file="overlay/overlay.go:559"
DEBU[2021-07-17 11:53:31.965836584Z] Overlay test mount with a single lower failed invalid argument  file="overlay/overlay.go:571"
ERRO[2021-07-17 11:53:31.966098625Z] 'overlay' is not supported over xfs at "/var/lib/crio/overlay"  file="overlay/overlay.go:573"
FATA[2021-07-17 11:53:31.966797714Z] Validating root config: failed to get store to set defaults: kernel does not support overlay fs: 'overlay' is not supported over xfs at "/var/lib/crio/overlay": backing file system is unsupported for this graph driver  file="crio/main.go:354"
```
The bug is related to https://github.com/containers/storage/issues/963 
It will be solved by bumping vendored go package github.com/containers/storage to at least `1.30.1`, which contains a check for existence of `"system_u:object_r:container_file_t:s0"` SELinux label in the kernel

**Output of `crio --version`:**

```
crio version 1.21.0
Version:       1.21.0
GitCommit:     bc1ef35a932acc2f6f3b6d3eb19a4f68aa9423f6
GitTreeState:  dirty
BuildDate:     1980-01-01T00:00:00Z
GoVersion:     go1.15.7
Compiler:      gc
Platform:      linux/amd64
Linkmode:      static
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
VM Flatcar Linux stable

**Comment by sdlarsen** [comment]
It's already bumped in master. Question is when we'll see a release with this... (@haircommander?)

**Comment by haircommander** [comment]
version bump PR happened https://github.com/cri-o/cri-o/pull/5085 so I just need to package it. I hope to do so today

**Comment by haircommander** [comment]
fix should be in 1.21.2, which should be packaged now in OBS

**Comment by sdlarsen** [comment]
1.21.2 works for me. Thanks! Would be nice if the releaset was visible here on github as well...

**Comment by haircommander** [comment]
done https://github.com/cri-o/cri-o/releases/tag/v1.21.2 :)

**Comment by haircommander** [comment]
fixed in 1.21.2 and beyond
